name-emu/src/mips.rs

```diff
@@ -0,0 +1,371 @@
+use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use std::fmt;
+use std::io::Cursor;
+
+pub const DOT_TEXT: u32 = 0x00400000;
```

Comment:
Maybe `DOT_TEXT_START_ADDRESS`?

Reply:
or instead of `DOT_TEXT...`, `TEXT_SECTION_....`

Reply:
done